src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,12 +19,46 @@
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all applications";
+    public static final String COMMAND_ORDER_WORD_ASCENDING = "ASC";
+
+    public static final String COMMAND_ORDER_WORD_DESCENDING = "DESC";
+
+    public static final String MESSAGE_SUCCESS = "Listed filtered applications";
+    // TODO: Update it to variable as per edit
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Show a sorted list of all applications with the given sort parameter.\n"
+            + "Parameters: [parameter "
+                + NameComparator.COMMAND_WORD.toUpperCase() + "|"
+                + InterviewSlotComparator.COMMAND_WORD.toUpperCase() + "|"
+                + PriorityComparator.COMMAND_WORD.toUpperCase() + "] "
+            + "[order " + COMMAND_ORDER_WORD_ASCENDING + "|" + COMMAND_ORDER_WORD_DESCENDING + "] \n"
+            + "Example: " + COMMAND_WORD + " INTERVIEW " + COMMAND_ORDER_WORD_DESCENDING + "\n"
+            + "Note: Using list without parameters will revert to the last sorted.";
 
+    private final Optional<Comparator<Application>> sortingComparator;
+    private final String orderBy;
+
+    /**
+     * Creates a ListCommand with the previous sorting order.
+     */
+    public ListCommand() {
+        sortingComparator = Optional.empty();
+        orderBy = COMMAND_ORDER_WORD_ASCENDING;
```

Comment:
Does this mean that by Default it will always display in ascending order? If so maybe you can indicate some where in the JavaDoc as well

Reply:
just placeholder, updated javadoc

Reply:
Nice summarized JavaDoc!